transform/mattermost-analytics/models/marts/product/licenses/_product__licenses__models.yml

```diff
@@ -91,3 +91,55 @@ models:
         description: |
           Whether the expiration date is consistent in case there's multiple sources that report the license. True if 
           more than one expiration dates are reported, false else.
+
+  - name: dim_self_hosted_customers
+    description: |
+      List of known self-hosted customers with active or inactive licenses along with their server details. 
+      Note that the same customer/server might appear more than once in case the details are different for the given license.
```

Comment:
This can introduce duplicates if the user doesn't know how this affects the join with the fact table. Please check the comment in the fact table for more details.